workspaces/ui/scripts/fetch-docs.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require('node-fetch');
```

Comment:
@notnmeyer thoughts?
@acunniffe I think this should be done more statically

Reply:
this is currently what we do for the saas site, but open to making it more static :)